src/data/git-quiz.ts

```diff
@@ -477,7 +477,7 @@ const gitQuiz = [
   {
     Question: "In Git, how do you apply a stash if you have multiple?",
     Answer: "git stash pop stash@{x}",
-    Distractor1: "git stash pop",
+    Distractor1: "Git stash pop",
```

Comment:
this change is extraneous for this PR, please remove it

Reply:
Removed.